Dockerfile

```diff
@@ -149,6 +149,8 @@ COPY --from=runtime-collect / /
 
 FROM ubuntu:24.04 AS test
 ARG TARGETARCH
+LABEL org.opencontainers.image.url="https://hub.docker.com/r/rancher/rke2-runtime"
```

Comment:
You've added these to the `test` target which is not published. I think you probably want to move them to the `runtime` step a couple lines up, which is the step that builds the actual published rke2-runtime linux image.

Reply:
Thank you for point it out, it was not very obvious when I first looked at the dockerfile